src/app/layout/notification-bar/dynamic-notifications/get-started-notification/get-started-notification.component.scss

```diff
@@ -0,0 +1,12 @@
+:host {
+    .open-project-btn {
+        background: none !important;
+        color: inherit;
+        border: none;
+        padding: 0 ! important;
+        font: inherit;
+        cursor: pointer;
+        outline: inherit !important;
+        text-decoration: underline;
+    }
```

Comment:
You can remove everything except `text-decoration` and `background`, add `vertical-align: initial` to `.btn.btn-unstyled` specification, then use that.
Or you can update global `.btn-link` to include style resets from `.btn-unstyled` + underline + different color, since that seems to be more intuitive.